src/_gettsim_tests/test_data/grundrente_proxy_rente/2021-07-01/hh_id_10.yaml

```diff
@@ -67,4 +71,4 @@ inputs:
   assumed: {}
 outputs:
   rente_vorj_vor_grundr_proxy_m:
-    - 1025.7
+    - 0.0
```

Comment:
I changed this because in the policy environment of 2021 retirement was not possible
with age 65 (because after the requested changes, GETTSIM "forgets" about the Rente wg.
Arbeitslosigkeit after 2017).